### PR TITLE
fix: RNN pass in decoder + loss computation

### PR DESCRIPTION
**Pull request by jannisborn**
Fixing https://github.com/GT4SD/gt4sd-core/issues/176

Once we merge this PR, we can release a new tag and make `gt4sd` dependent on it. This should resolve the issue.

### CODE REVIEW

File: moses/vae/model.py [hunk 1]
```diff
@@ -123,10 +123,6 @@ def forward_decoder(self, x, z):
 
         x = nn.utils.rnn.pad_sequence(x, batch_first=True,
                                       padding_value=self.pad)
-        x_emb = self.x_emb(x)
```
Comment (davidegraff) [inline moses/vae/model.py:126]: why delete this line? Without it, you're passing in the raw token indices to your RNN

Comment (jannisborn) [reply moses/vae/model.py:126]: You're totally right, should be fixed now. Thanks for checking!

Comment (jannisborn) [reply moses/vae/model.py:126]: unless you have any other comment @davidegraff, I will merge this later, release the new tag and prepare the PR for the gt4sd-core

**Review by davidegraff — APPROVED** [review]
looks good from my end

**Pull request merged**